api/src/main/java/io/kafbat/ui/controller/ApplicationConfigController.java

```diff
@@ -46,6 +48,12 @@ interface PropertiesMapper {
     DynamicConfigOperations.PropertiesStructure fromDto(ApplicationConfigPropertiesDTO dto);
 
     ApplicationConfigPropertiesDTO toDto(DynamicConfigOperations.PropertiesStructure propertiesStructure);
+
+    default ActionDTO stringToActionDto(String str) {
+      return Optional.ofNullable(str)
+          .map(s -> Enum.valueOf(ActionDTO.class, s.toUpperCase()))
```

Comment:
That fixes the case mismatch but doesn't fix the other case from the issue for "all" value which is acceptable on config level but it's missing from the enum as we map the values in runtime

Reply:
fixed